zulipterminal/config/themes.py

```diff
@@ -52,11 +52,7 @@ def generate_theme(theme_name: str, color_depth: int) -> ThemeSpec:
         bg_codes = bg.value.split()
         new_style: Tuple[Optional[str], ...] = tuple()
         if color_depth == 1:
-            # FIXME: Check for completeness of REQUIRED_STYLES
-            try:
-                new_style = (style_name, "", "", REQUIRED_STYLES[style_name])
-            except KeyError:
-                continue
```

Comment:
If the test migration was earlier this change might not be necessary.

That said, we will want to add more error handling to this function later, for situations where the code is updated without running the tests (ie. user modification/exploration of themes) - so you could just leave this in I think?

Reply:
Once we check for equality, this case is covered and shows a incomplete warning. Maybe we can change the wording and explain what incomplete means -
`Warning: Incomplete theme! (either STYLES or REQUIRED_STYLES has missing styles)`?

Reply:
I'm removing try-except for now. Maybe later when we need other error handling we can add them?

Reply:
Do I change the wording for the mentioned Warning?